components/Cards/src/MDCCard.h

```diff
@@ -113,4 +115,10 @@
  */
 - (nullable UIColor *)shadowColorForState:(UIControlState)state UI_APPEARANCE_SELECTOR;
 
+/*
+ The shape generator used to define the card's shape.
```

Comment:
If this is null will the card behave differently in any way? Document if so.

Reply:
Please see documentation update below